### ISSUE TITLE
SmartCity-Accessibility Server Side Database Branch #10 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#267]
Build 'SmartCity-Accessibility Server Side Database Branch' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [Alexander Kaplan] 687399059379ff64aa3810a041cb27754b138fb8 - added test category for reveiwmanagertest

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Database%20Branch/10/)

**Comment by JuanJenkinsBot** [comment]
Error processing tokens: Error while parsing action 'Text/ZeroOrMore/FirstOf/Token/DelimitedToken/DelimitedToken_Action3' at input position (line 10, pos 102):
${BUILD_LOG_EXCERPT,start="*-------------------------------------------------------*",end="*[INFO]*"}
                                                                                                     ^

java.util.regex.PatternSyntaxException: Dangling meta character '*' near index 0
*-------------------------------------------------------*
^

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Database Branch' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [Alexander Kaplan] 687399059379ff64aa3810a041cb27754b138fb8 - added test category for reveiwmanagertest

No changes
- [Alexander Kaplan] f5de314c82463923c532b1ca8bfa976ab432fcb2 - updated maven compiler pluging as a hail mary



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Database%20Branch/12/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Database Branch' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [Alexander Kaplan] 687399059379ff64aa3810a041cb27754b138fb8 - added test category for reveiwmanagertest

No changes
- [Alexander Kaplan] f5de314c82463923c532b1ca8bfa976ab432fcb2 - updated maven compiler pluging as a hail mary

No changes


[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Database%20Branch/13/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Database Branch' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [Alexander Kaplan] 687399059379ff64aa3810a041cb27754b138fb8 - added test category for reveiwmanagertest

No changes
- [Alexander Kaplan] f5de314c82463923c532b1ca8bfa976ab432fcb2 - updated maven compiler pluging as a hail mary

No changes
No changes


[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Database%20Branch/14/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
